plugins/aglight/plugin.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "../../schemas/plugin.json",
+
+    "name": "AGLight v1.0.0",
+    "previousVersions": {},

```

Comment:
You can remove this line

Reply:
The `previousVersions` or the blank line?

Reply:
`previousVersions` is not needed when it's empty.